README.md

```diff
@@ -99,6 +99,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2020/edit/mas
 |[Roman Rakhlin](https://www.instagram.com/rrakhlin/)|[GitHub](https://github.com/romarakhlin/WWDC20-Submission)|[Youtube](https://www.youtube.com/watch?v=i3y5k_khW_I)|UIKit, SceneKit, SpriteKit|Submitted|
 |[Robert Pliev](https://github.com/camotsuc)|[GitHub](https://github.com/camotsuc/wwdc20ChallengeAttempt)||UIKit, Foundation|Rejected|
 |[Roy Rao](https://github.com/RoyRao2333)|[GitHub](https://github.com/RoyRao2333/WWDC20-Scholarship)||Cocoa, Security, Playground Support|Submitted|
+|[Rubén Fernández](https://twitter.com/RubenApps)|[GitHub](https://github.com/Rubenfer)||SwiftUI|Accepted|
```

Comment:
You linked to your GitHub profile and not your project in the GitHub project repository

Reply:
For the moment I don't have the project available in GitHub. I've removed the GitHub section.